EIPS/eip-3076.md

```diff
@@ -3,11 +3,12 @@ eip: 3076
 title: Slashing Protection Interchange Format
 author: Michael Sproul (@michaelsproul), Sacha Saint-Leger (@sachayves), Danny Ryan (@djrtwo)
 discussions-to: https://ethereum-magicians.org/t/eip-3076-validator-client-interchange-format-slashing-protection/
-status: Review
+status: Last Call
 type: Standards Track
-category: Core
+category: Interface
 created: 2020-10-27
-updated: 2021-02-08
+updated: 2021-09-20
+review-period-end: 2021-10-20
 ---
 
 ## Simple Summary
```

Comment:
Simple summary section has been removed, and in its place a `description` header field has been added.  Please delete this section and add a frontmatter line item above like:
```md
description: A JSON interchange format for proof of stake validators to migrate slashing protection data between clients.
```

Reply:
Thanks, done.